indico/modules/users/models/export.py

```diff
@@ -0,0 +1,133 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2023 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import ARRAY
+
+from indico.core.db import db
+from indico.core.db.sqlalchemy import PyIntEnum, UTCDateTime
+from indico.util.date_time import now_utc
+from indico.util.enum import RichIntEnum
+from indico.util.i18n import L_
+
+
+class DataExportRequestState(RichIntEnum):
+    __titles__ = [L_('Pending'), L_('Running'), L_('Success'), L_('Failed'), L_('Expired')]
+    pending = 0  # Default value when there is no request
+    running = 1
+    success = 2
+    failed = 3
+    expired = 4  # The associated file has been deleted
+
+
+class DataExportOptions(RichIntEnum):
+    __titles__ = [L_('Personal data'), L_('Settings'), L_('Minutes & Contributions'), L_('Registrations'),
+                  L_('Room booking data'), L_('Abstracts & Papers'), L_('Survey submissions'),
+                  L_('Attachments & Materials'), L_('Editables'), L_('Miscellaneous')]
+    personal_data = 0
```

Comment:
I usually try to avoid `0` for enum members that do not have some falsy meaning and start with 1 there (you can just put a `None` as the first titles item in that case)